deb/common/control

```diff
@@ -102,6 +103,14 @@ Description: Rootless support for Docker.
   Either VPNKit or slirp4netns (>= 0.4.0) needs to be installed separately.
 Homepage: https://docs.docker.com/engine/security/rootless/
 
+Package: docker-buildx-plugin
+Priority: optional
+Replaces: docker-ce-cli
```

Comment:
I initially was confused by the `Replaces:` here, as I thought it would mark this package to replace the whole `docker-ce-cli` package, but @crazy-max explained this also can be used to replace _some files_ (given, it's a bit "fuzzy", as it doesn't require speciyfing _which_ files), from the [The Debian Administrator's Handbook](https://debian-handbook.info/browse/stable/sect.package-meta-information.html#id-1.8.6.7.14):


> ### 5.2.1.5. Replacing Files: The Replaces Field

> The Replaces field indicates that the package contains files that are also present in another package, but that the package is legitimately entitled to replace them. Without this specification, dpkg fails, stating that it can not overwrite the files of another package (technically, it is possible to force it to do so with the --force-overwrite option, but that is not considered standard operation). This allows identification of potential problems and requires the maintainer to study the matter prior to choosing whether to add such a field.
>
>The use of this field is justified when package names change or when a package is included in another. This also happens when the maintainer decides to distribute files differently among various binary packages produced from the same source package: a replaced file no longer belongs to the old package, but only to the new one.
>
> If all of the files in an installed package have been replaced, the package is considered to be removed. Finally, this field also encourages dpkg to remove the replaced package where there is a conflict.

The last paragraph outlines our situation

What we probably _could_ consider doing is (if supported) to include the version here, e.g.

```suggestion
Replaces: docker-ce-cli (<< 22.04.0)
```

But we need to check if it likes CalVer; instead, probably this would work;

```suggestion
Replaces: docker-ce-cli (<= 21.10~)
```

Reply:
Unfortunately I tried and it doesn't work with CalVer

Reply:
About relationships see also https://www.debian.org/doc/debian-policy/ch-relationships.html#overwriting-files-and-replacing-packages-replaces